examples/wrappers/rr.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# Record deterministic execution using rr
+#
+
+export _RR_TRACE_DIR=$AVOCADO_TEST_LOGDIR/rr
+mkdir -p $_RR_TRACE_DIR
+exec rr record "$@"
```

Comment:
Yep, when I add `-n` it works fine. Maybe one note the test results is standardized and should not contain anything except the files mentioned in the documentation. How about putting the tracedir into `$AVOCADO_TEST_LOGDIR/data`?